Gruntfile.js

```diff
@@ -781,6 +781,25 @@ module.exports = function(grunt) {
           pattern: /console\./g,
         },
       },
+      'console-in-node': {
```

Comment:
Well `static-analysis` calls the `regex-check` task so It will iterate over all targets in `regex-check` including `console-in-node`. No need to add it